client/components/footer-section/index.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from '@emotion/styled';
+import { ReactChild } from 'react';
+
+interface FooterSectionProps {
+	header: ReactChild;
+	children: ReactChild | ReactChild[];
+}
+
+const FooterContainer = styled.div`
+	::before {
+		box-sizing: border-box;
+		content: '';
+		background-color: #fafafa;
```

Comment:
Can we use a variable here?

Reply:
Haven't found this variable, [it seems it is gray-98](https://www.color-hex.com/color/fafafa) should I add it to the color schemes for wider use or I can totally add a `style.css` file with the declaration at the component lvl?

I favor the first option!

Reply:
let's align with @SaxonF  / @vinimotaa  on that. We should either update the value of the color schemes OR use another shade that already exists there.